atom/browser/api/atom_api_window.cc

```diff
@@ -424,6 +424,16 @@ void Window::ShowDefinitionForSelection() {
 }
 #endif
 
+#if defined(OS_MACOSX) || defined(OS_LINUX)
```

Comment:
Should we need to expose the two js APIs `setVisibleOnAllWorkspaces` and `isVisibleOnAllWorkspaces` on Windows?
How about only restricting them in `BuildPrototype` function?